public/app/features/alerting/unified/api/onCallApi.ts

```diff
@@ -0,0 +1,27 @@
+import { alertingApi } from './alertingApi';
+export interface OnCallIntegration {
+  [key: string]: string;
+  integration_url: string;
+}
+export type OnCallIntegrationsResponse = OnCallIntegration[];
+export type OnCallIntegrationsUrls = string[];
+
+export const onCallApi = alertingApi.injectEndpoints({
+  endpoints: (build) => ({
+    getOnCallIntegrations: build.query<OnCallIntegrationsUrls, void>({
+      query: () => ({
+        headers: {},
+        url: '/api/plugin-proxy/grafana-oncall-app/api/internal/v1/alert_receive_channels/?search=',
+      }),
+      providesTags: ['OnCallIntegrations'],
+      transformResponse: (response: OnCallIntegrationsResponse) => response.map((result) => result.integration_url),
+    }),
+  }),
+});
+
+export const useGetOnCallIntegrations = (skip: boolean) => {
```

Comment:
This function feels a bit redundant, I think calling `onCallApi.useGetOnCallIntegrationsQuery()` where we need it (and just disregard loading and error if we're not going to be using it) works just fine.

Reply:
true!, fixed ✅